src/backend/mailbox_manager/tests/fixtures/dimail.py

```diff
@@ -80,3 +84,14 @@
         ],
     },
 }
+
+## TOKEN
+
+TOKEN_OK = json.dumps({"access_token": "token", "token_type": "bearer"})
+
+## MAILBOXES
+
+
+def response_mailbox_created(email_address):
```

Comment:
I'm unsure about this name. Caps or not caps ? WDYT @Morendil @sdemagny

Reply:
En fait c'est que, dans ma tête en majuscules c'est pour les constantes. Mais là je passe le nom du domaine en argument donc ça défit pas mal la définition de "constante", non ?

Reply:
Mince j'ai cru supprimer un doublon de commentaire qui s'affichait plus bas mais tout est parti. 
@mjeammet tu as raison. Je n'ai pas trop d'idée à chaud peut être le nommage. Dispo pour en discuter sinon.